trials/qa1/definition.json

```diff
@@ -7,32 +7,32 @@
     "spring_config_label": "main",
     "site_service": {
         "image_name": "ghcr.io/ndph-arts/site-service",
-        "image_tag": "0e310062adb3569db688214ef4ad7273a7918b4d"
```

Comment:
qa1, should not be changed. it should have a specific tag

Reply:
reverted